src/less/components/table.less

```diff
@@ -244,6 +244,7 @@
 
     .uk-table-responsive,
     .uk-table-responsive tbody,
+    .uk-table-responsove tfoot,
```

Comment:
typo `responsove` should be `responsive `

Reply:
Thanks, fixed!